### PR TITLE
Fix non-multiselectable AccordionPanel bug

### PR DESCRIPTION
**Pull request by mperrotti**
Re Prelim - thinking on whether this is the best solution

#### Related issues
Fixes https://meetup.atlassian.net/browse/WC-193

#### Description
Open `<AccordionPanel>`s collapse their height when AccordionPanelGroup is `multiselectable`. Moved setState stuff from `this._handleToggle` to `componentWillReceiveProps`

### CODE REVIEW

File: src/interactive/AccordionPanel.jsx [hunk 1]
```diff
@@ -34,6 +34,9 @@ class AccordionPanel extends React.Component {
 	getPanelStyle(isOpen, contentEl) {
 		const style = { height: '0px' };
 
+		// console.log(`isOpen: ${isOpen}`);
```
Comment (akdetrick) [inline src/interactive/AccordionPanel.jsx]: i recommend deleting these instead of commenting them out

Comment (mperrotti) [reply src/interactive/AccordionPanel.jsx]: woops, missed that one

File: src/interactive/AccordionPanel.jsx [hunk 2]
```diff
@@ -60,17 +63,7 @@ class AccordionPanel extends React.Component {
...
-			() => {
-				setTimeout(() => {
-					this.setState(()=>
-						this.getPanelStyle(!isOpen, this.contentEl)
-					);
-				}, 1);
-			}
-		);
-		setClickedPanel && setClickedPanel(this.props.clickId, !isOpen);
+		setClickedPanel && setClickedPanel(this.props.clickId, !isOpen, this.getPanelStyle(!isOpen, this.contentEl)); // this.getPanelStyle(!isOpen, this.contentEl)
```
Comment (akdetrick) [inline src/interactive/AccordionPanel.jsx]: let's remove this comment too

File: src/interactive/AccordionPanel.jsx [hunk 3]
```diff
@@ -88,6 +81,25 @@ class AccordionPanel extends React.Component {
...
 
+	/**
+	 * @description set height state when the panel receives a change
+	 * @returns {undefined}
+	 */
+	componentWillReceiveProps(nextProps) {
+		if (nextProps.isOpen !== this.props.isOpen) {
+			this.setState(()=>
+				this.getPanelStyle(!nextProps.isOpen, this.contentEl),
+				() => {
```
Comment (akdetrick) [inline src/interactive/AccordionPanel.jsx]: this extra anonymous function may not be necessary; I believe you should be able to directly call the `setTimeout` in the `setState` callback

Comment (mperrotti) [reply src/interactive/AccordionPanel.jsx]: I removed it and it broke. Keeping it in

**Pull request merged**